src/WorkflowManager/WorkflowManager/Controllers/WorkflowInstanceController.cs

```diff
@@ -139,14 +141,57 @@ public async Task<IActionResult> GetByIdAsync([FromRoute] string id)
                 {
                     _logger.LogDebug($"{nameof(GetByIdAsync)} - Failed to find workflow instance with Id: {id}");
 
-                    return Problem($"Failed to find workflow instance with Id: {id}", $"/workflows/{id}", NotFound);
+                    return Problem($"Failed to find workflow instance with Id: {id}", $"{ENDPOINT}{id}", NotFound);
                 }
 
                 return Ok(workflowInstance);
             }
             catch (Exception e)
             {
-                return Problem($"Unexpected error occured: {e.Message}", $"/workflowinstances/{nameof(id)}", InternalServerError);
+                return Problem($"Unexpected error occurred: {e.Message}", $"{ENDPOINT}{nameof(id)}", InternalServerError);
+            }
+        }
+
+        /// <summary>
+        /// Returns failed workflow instances, that have an empty value of.
+        /// </summary>
+        /// <param name="acknowledged">Failed workflow's since this date.</param>
+        /// <returns>This should be a new endpoint to return failed workflow's instances, that have an empty value of.</returns>
+        [Route("failed")]
+        [HttpGet]
+        public async Task<IActionResult> GetFailedAsync([FromQuery] string acknowledged)
+        {
+            if (string.IsNullOrWhiteSpace(acknowledged))
+            {
+                return Problem($"Failed to validate, no {nameof(acknowledged)} parameter provided", $"{ENDPOINT}failed", BadRequest);
+            }
+
+            var parseResult = DateTime.TryParse(acknowledged, out var acknowledgedDateTime);
```

Comment:
Should we use an [IFormatProvider](https://learn.microsoft.com/en-us/dotnet/api/system.datetime.tryparse?view=net-7.0#system-datetime-tryparse(system-string-system-iformatprovider-system-globalization-datetimestyles-system-datetime@)) here? Would this work for other date formats in different cultures?

Reply:
This bit of code has been refactored so ASP.net handles parsing and in params has a bit of text specifiy it should be passed in as ISO 8601 standard YYYY-MM-DD, even YYYY-MM-DDT00:00 will work.